.kokoro/psm_interop_kokoro_lib.sh

```diff
@@ -694,6 +739,10 @@ activate_gke_cluster() {
       GKE_CLUSTER_NAME="interop-test-psm-basic"
       GKE_CLUSTER_ZONE="us-central1-c"
       ;;
+    GKE_CLUSTER_DUAL_STACK)
+          GKE_CLUSTER_NAME="psm-interop-lb-primary-ds"
+          GKE_CLUSTER_ZONE="us-central1-a"
+          ;;
```

Comment:
```suggestion
    GKE_CLUSTER_DUAL_STACK)
      GKE_CLUSTER_NAME="psm-interop-lb-primary-ds"
      GKE_CLUSTER_ZONE="us-central1-a"
      ;;
```

The indentation is off - please fix